python/lsst/daf/butler/cli/butler.py

```diff
@@ -148,24 +199,14 @@ def get_command(self, context, name):
         click.Command
             A Command that wraps a callable command function.
         """
-        name = cmdNameToFuncName(name)
-        localCmd = self._getLocalCommand(name)
-        if localCmd is not None:
-            return localCmd
-        for pluginName in self._getPluginList():
-            plugin = self._importPlugin(pluginName)
-            if plugin is None:
-                continue
-            for command in plugin.__all__:
-                if command == name:
-                    fullCommand = pluginName + "." + command
-                    try:
-                        cmd = doImport(fullCommand)
-                    except Exception as err:
-                        log.debug("Command import exception: %s", err)
-                        context.fail("Could not import command {fullCommand}")
-                    return cmd
-        return None
+        if self.commands is None:
+            self.commands = self._getCommands()
+        if name not in self.commands:
+            return None
+        self._raiseIfDuplicateCommands(self.commands)
+        if self.commands[name][0] == "lsst.daf.butler.cli.cmd":
```

Comment:
Please set this as a constant somewhere since you have used it twice (you could even consider deriving it from `__name__` rather than hardcoding it).